experiments/Cargo.toml

```diff
@@ -29,7 +29,12 @@ jexl-eval = "0.1.6"
 uuid = { version = "0.8", features = ["serde", "v4"]}
 sha2 = "0.9"
 hex = "0.4"
+# We use a fixed version of ffi-support to resolve the conflict between viaduct and uniffi in the versions of ffi_support they use
```

Comment:
Hmmm looks like this issue is gone.... Not 100% sure why it's gone, but my guess is the Cargo.lock is helping us out here 😄

I don't think we should need to match the versions and it's weird that the issue showed up in the first place, since viaduct uses version "0.4" and uniffi uses "~0.4.2". I would naively expect cargo to know to pick "0.4.2"... Anywho, not an issue anymore